samples/package.json

```diff
@@ -14,6 +14,8 @@
     "test": "mocha system-test --timeout 600000"
   },
   "dependencies": {
+    "@opentelemetry/api": "^0.10.2",
+    "@opentelemetry/tracing": "^0.10.2"
```

Comment:
Ah @sethmaxwl it looks like this is what the linter is complaining about. There are now two lines in a row with no comma.